doc/en/writing_plugins.rst

```diff
@@ -157,6 +158,10 @@ it in your setuptools-invocation:
         classifiers=["Framework :: Pytest"],
     )
 
+Or via tool specific mechanism when using other packaging framework like `poetry
+<https://python-poetry.org/docs/pyproject/#plugins>`_ or `flit
+<https://flit.readthedocs.io/en/stable/entrypoints.html#specifying-entry-points>`_.
```

Comment:
I think this is not correct, these tools have their own settings for generating setuptools entry_points

Reply:
Well if you can register, and list, entry points without setuptools on your system, are they setuptools entry_points ? 

I guess it's a question of semantic, yes entry_point of this type have been initially created by setuptools sure they are "setuptools entry_points" the problem is that if you keep referring to them as "setuptools entry_points" user will assume you _need_ setuptools to create those.

Reply:
afaik all of the example tools added in this PR use setuptools under the hood to create them so...

Reply:
I didn't think flit used setuptools to make wheels

Reply:
Yes that is correct, I send the PR as I created https://github.com/Carreau/pytest_covid/blob/master/pyproject.toml using flit, that does works well, and does not use setuptools.

Reply:
nah, it definitely uses setuptools, here's the code for it -- ^F entry_points: https://github.com/takluyver/flit/blob/master/flit/sdist.py

Reply:
That's the sdist shim for build systems without PEP517/518 here's how it works when creating whl:

https://github.com/takluyver/flit/blob/master/flit_core/flit_core/wheel.py#L193

Reply:
Yes. You _can_ generate a `setup.py` for compatibility but don't have to. So definitively no you do not have to use setuptools to generate entry_points.

Reply:
ah I see flit has changed, either way the idea of entry points was "standardized" by setuptools so I believe it is correct to call them "setuptools entrypoints"

Reply:
Seems they're defined here now: https://packaging.python.org/specifications/entry-points/

Reply:
> The entry point file format was originally developed to allow packages built with setuptools to provide integration point metadata that would be read at runtime with pkg_resources. It is now defined as a PyPA interoperability specification in order to allow build tools other than setuptools to publish pkg_resources compatible entry point metadata, and runtime libraries other than pkg_resources to portably read published entry point metadata (potentially with different caching and conflict resolution strategies).

Reply:
Is there a list of other PyPA interoperability specifications? Or is it really just defacto pip implementation defined?

Reply:
IMHO opinion we should dwell too much into the semantics here: I think the most important message is that users are able to create plugins using a myriad of tools: setup.py/setuptools, flint, etc.